core/module/Dog/dog_modules/dog_module/Shadowlamb/cmd/info/motd.php

```diff
@@ -4,14 +4,12 @@ final class Shadowcmd_motd extends Shadowcmd
 	public static function execute(SR_Player $player, array $args)
 	{
 		$bot = Shadowrap::instance($player);
-		$path = Shadowrun4::getShadowDir().'dog_module/Shadowlamb/shadowlamb_motd.txt';
-		if (false === ($motd = @file_get_contents($path)))
+		$path = Shadowrun4::getShadowDir().'shadowlamb_motd.txt';
+		if (!is_readable($path) || false === ($motd = file_get_contents($path)))
 		{
-			$bot->reply('Can not read '.$path);
+			return $bot->reply('Can not read MOTD file');
 		}
 		return $bot->rply('5249', array($motd));
-// 		$bot->reply(sprintf('Message of the day: %s', $motd));
```

Comment:
We have a lot of these comments, imho they help a little bit to find out which string translation '5249' is.

Reply:
> We have a lot of these comments, imho they help a little bit to find out which string translation '5249' is.

Not sure this is the correct way to solve an issue with translations being unclear, but it's the fast way without changing the whole translation system ;)
Brought that comment back